examples/coexist_klog_v1_using_logr/coexist_klog_v1.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"flag"
+	"github.com/go-logr/logr"
+
+	"k8s.io/klog"
+	klogv2 "k8s.io/klog/v2"
+)
+
+type klogLogger struct{}
+
+func (kw klogLogger) Info(msg string, keysAndValues ...interface{}) {
+	// if you start using structured logging, please serialize as well
+	klog.InfoDepth(klogv2.LoggerCallDepth, msg)
+}
+
+func (kw klogLogger) Enabled() bool {
+	return true
+}
+
+func (kw klogLogger) Error(err error, msg string, keysAndValues ...interface{}) {
+	// if you start using structured logging, please serialize as well
+	klog.ErrorDepth(klogv2.LoggerCallDepth, msg)
+}
+
+func (kw klogLogger) V(level int) logr.InfoLogger {
+	// tweak the return value based in the level as needed
+	return kw
+}
+
+func (kw klogLogger) WithValues(keysAndValues ...interface{}) logr.Logger {
+	// if you start using structured logging, please serialize as well
+	return kw
+}
+
+func (kw klogLogger) WithName(name string) logr.Logger {
+	// if you start using structured logging, please serialize as well
+	return kw
+}
+
+func main() {
+	klog.InitFlags(nil)
+	klogv2.LogToStderr(false)
+	klogv2.SetLogger(klogLogger{})
```

Comment:
similar question... only the klogv2 lines are relevant for someone who already has klog v1 set up as desired?